controller/eks-cluster-config-handler.go

```diff
@@ -1020,21 +1074,48 @@ func (h *Handler) updateUpstreamClusterState(upstreamSpec *eksv1.EKSClusterConfi
 		// happen together
 
 		ng := ngs[aws.StringValue(upstreamNg.NodegroupName)]
```

Comment:
Did you test updating the launch template from both sides ? The CR and the AWS console.

Reply:
Yep.